crates/database/src/events.rs

```diff
@@ -68,13 +68,21 @@ pub async fn delete(
     ex.execute(sqlx::query(QUERY_SETTLEMENTS).bind(delete_from_block_number))
         .await?;
 
+    const QUERY_PRESIGNATURES: &str = "DELETE FROM presignature_events WHERE block_number >= $1;";
+    ex.execute(sqlx::query(QUERY_PRESIGNATURES).bind(delete_from_block_number))
+        .await?;
+
+    // Observations and order executions are not events but data deducted from the
```

Comment:
The word should probably be deduced or derived.

Reply:
fixed